src/styles/styles.js

```diff
@@ -776,6 +776,54 @@ const styles = {
         justifyContent: 'center',
     },
 
+    emojiPickerContainer: {
+        backgroundColor: themeColors.componentBG,
+        minWidth: 360,
+    },
+
+    emojiPickerList: {
+        height: 300,
+        width: '100%',
+        paddingHorizontal: 16,
+    },
+
+    emojiHeaderStyle: {
+        backgroundColor: themeColors.componentBG,
+        width: '100%',
+        paddingVertical: 12,
+        fontFamily: fontFamily.GTA_BOLD,
+        fontWeight: fontWeightBold,
+        color: themeColors.heading,
+        fontSize: variables.fontSizeSmall,
+    },
+
+    // Emoji Picker Styles
+    emojiText: {
+        fontFamily: fontFamily.GTA_BOLD,
+        fontSize: variables.iconSizeLarge,
+        paddingTop: 4,
+        paddingBottom: 4,
+        paddingRight: 8,
+        paddingLeft: 8,
+    },
+
+    emojiItem: {
+        width: '12.5%',
+        textAlign: 'center',
+    },
+
+    chatItemEmojiButton: {
+        alignSelf: 'flex-end',
+        borderRadius: 6,
+        height: 32,
+        paddingTop: 6,
```

Comment:
I think you can just do `padding: 6` instead of values for each.

Reply:
Also, just general feedback on this is that we've tried to be pretty consistent about doing spacing in increments of 4 throughout the app, so I'd encourage you to try to keep to that. If it looks bad of course NAB but I think it's worth looking into.

i.e:
`padding: 6` ---> `...spacing.p2`
`margin: 3` ---> `...spacing.m1`

Reply:
We probably don't even need padding on this button if we just center align the icon inside of it both vertically and horizontally.

Reply:
I think we do need the odd margin number though.